packages/@angular/cli/blueprints/ng/files/__path__/app/app.component.html

```diff
@@ -1,4 +1,20 @@
-<h1>
-  {{title}}
-</h1><% if (routing) { %>
+<div style="text-align:center">
+  <h1>
+    Welcome to {{title}}!!
+  </h1>
+  <img width="300" src="https://angular.io/resources/images/logos/angular/angular.svg" />
```

Comment:
Could you inline the image? It's SVG so you can just copy its content in the HTML, and that way even without a network connection you get it.